src/sentry/api/serializers/rest_framework/release.py

```diff
@@ -15,10 +15,38 @@ class ReleaseHeadCommitSerializerDeprecated(serializers.Serializer):
 
 
 class ReleaseHeadCommitSerializer(serializers.Serializer):
-    commit = serializers.CharField(max_length=MAX_COMMIT_LENGTH)
+    commit = serializers.CharField()
```

Comment:
I'm guessing the downstream model code knows how to split a commit string containing `..` into its prev/next parts?

Reply:
Yup, right here: https://github.com/getsentry/sentry/blob/75ab9e5e3f254dfdffb133096eeb158b525499d9/src/sentry/models/release.py#L275-L288